jenkins/slaves/roles/php/tasks/main.yml

```diff
@@ -62,7 +62,7 @@
   with_items: '{{php_releases}}'
 
 - name: Adjust php memory limit
-  shell: echo "memory_limit = 512M" >> "{{ phpenv_versions_path }}/{{ item.version }}/etc/php.ini"
+  shell: echo "memory_limit = -1" >> "{{ phpenv_versions_path }}/{{ item.version }}/etc/php.ini"
```

Comment:
> Pro request: there is an ansible command to replace a line in a file. This would be better

http://docs.ansible.com/ansible/lineinfile_module.html